content/contribute/elasticsearch-develop.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "ElasticSearch notifier development"
+weight: 11
+---
+
+To develop BotKube with Elasticsearch notifier enabled, install it locally:
+
+1. Install Elasticsearch:
+
+    ```bash
+    helm repo add elastic https://helm.elastic.co
+    helm install elasticsearch elastic/elasticsearch --set replicas=1 --set resources.requests.cpu="100m" --set resources.requests.memory="512M" --wait
```

Comment:
```suggestion
    helm repo add elastic https://helm.elastic.co
    helm repo update elastic
    helm install elasticsearch elastic/elasticsearch --set replicas=1 --set resources.requests.cpu="100m" --set resources.requests.memory="512M" --wait
```

Reply:
I don't think that's needed if you add a new repo to Helm, but I agree that it might be handy to do update if someone already has this repo added 👍

Reply:
yes, that's the purpose of it, and I saw that in all the official docs, it's looks like a good practice 👍 

Elastic also has that: https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-install-helm.html

Reply:
I know that it's in the official docs, but still I didn't see that very often. For example in Argo: https://github.com/argoproj/argo-helm/tree/main/charts/argo-cd#installing-the-chart or bitnami: https://github.com/bitnami/charts - but anyway, as `helm repo add` exits with 0 if a given repo is already added to local config, it totally makes sense to have it to make sure it works even if repo is added 👍